src/base/bittorrent/torrenthandle.h

```diff
@@ -88,6 +88,9 @@ namespace BitTorrent
         // for restored torrents
         qreal ratioLimit;
         int seedingTimeLimit;
+#ifdef Q_OS_WIN
+        bool needSaveResumeData;
```

Comment:
You can just assign the default `false` here. It is better practice IMO.

Reply:
I am following what is being done on the rest of the struct. I don't want to refactor it now. Or leave it half and half.